tensorboard/plugins/interactive_inference/README.md

```diff
@@ -316,3 +316,15 @@ For TensorFlow GPU support, use the witwidget-gpu package instead of witwidget.
 
 Then, use it as seen at the bottom of the
 [What_If_Tool_Notebook_Usage.ipynb notebook](https://colab.research.google.com/github/tensorflow/tensorboard/blob/master/tensorboard/plugins/interactive_inference/What_If_Tool_Notebook_Usage.ipynb).
+
+### How do I enable it for use in a JupyterLab or Cloud AI Platform notebook?
+Install and enable WIT for JupyterLab by running a cell containing:
+```
+!pip install witwidget
+!jupyter labextension install wit-widget
```

Comment:
Check out npm [postinstall scripts](https://docs.npmjs.com/misc/scripts#examples), it _might_ be able to do these steps. (but your script will need to be compatible with all the environments it could run—or check which environment it is in).

Reply:
i'm going to leave this as-is for now, as in some environments it may need to be done with sudo (cloud ai platform notebooks), but not in your own jupyterlab instance.